libs/resolver/workspace.rs

```diff
@@ -835,30 +835,23 @@ impl<TSys: FsMetadata + FsRead> WorkspaceResolver<TSys> {
             })
           {
             // no configs have an import map and none are a package, so exit
-            return Ok(None);
+            return Ok(vec![]);
           }
 
-          let config_specified_import_map = match root_deno_json.as_ref() {
-            Some(deno_json) => deno_json
-              .to_import_map_value(sys)
-              .map_err(|source| WorkspaceResolverCreateError::ImportMapFetch {
-                referrer: deno_json.specifier.clone(),
-                source: Box::new(source),
+          let config_specified_import_maps = match root_deno_json.as_ref() {
+            Some(deno_json) => {
+              deno_json.to_import_map_values(sys).map_err(|source| {
+                WorkspaceResolverCreateError::ImportMapFetch {
+                  referrer: deno_json.specifier.clone(),
+                  source: Box::new(source),
+                }
               })?
-              .unwrap_or_else(|| {
-                (
-                  Cow::Borrowed(&deno_json.specifier),
-                  serde_json::Value::Object(Default::default()),
-                )
-              }),
-            None => (
-              Cow::Owned(workspace.root_dir_url().join("deno.json").unwrap()),
-              serde_json::Value::Object(Default::default()),
-            ),
+            }
+            None => vec![],
           };
           let base_import_map_config = import_map::ext::ImportMapConfig {
-            base_url: config_specified_import_map.0.into_owned(),
-            import_map_value: config_specified_import_map.1,
+            base_url: workspace.root_dir_url().join("deno.json").unwrap(),
```

Comment:
There was likely a bug: When the workspace root had import_map specified, all the member imports seem to be resolved relative to the root of the import_map, instead of the workspace root itself, now it is always resolved to workspace root